ViAn/GUI/zoompreviewwidget.cpp

```diff
@@ -0,0 +1,69 @@
+#include "zoompreviewwidget.h"
+#include <QDebug>
+
+/**
+ * @brief zoompreviewwidget::center_image
+ * Centers the image in the widget
+ * @param s QSize   : The size of the image to be centered
+ */
+void ZoomPreviewWidget::center_image(const QSize &s) {
+    // Will center the image both horizontally and vertically
+    // anchor = QPoint((this->width() - s.width()) / 2, (this->height()- s.height()) / 2);

```

Comment:
Outcommented line of code?

Reply:
I saved that because I wasn't sure if we wanted to center the widget vertically as well. It looks a bit weird if the preview widget is the only one in a dock area. @niklasniklas or @klwing021 might be able to answer that?

Reply:
@fredrikiselius: I would prefer centered both horizontal and vertical.

Reply:
Other than that. Ok to merge (after solving conflicts)